src/main/java/frc/robot/RobotContainer.java

```diff
@@ -61,7 +65,10 @@ public RobotContainer() {
    * PS4} controllers or {@link edu.wpi.first.wpilibj2.command.button.CommandJoystick Flight
    * joysticks}.
    */
-  private void configureBindings() {}
+  private void configureBindings() {
+
+    new JoystickButton(m_driverController, XboxController.Button.kLeftBumper.value).whileTrue(new InstantCommand(() -> intake.toggleIntake(), intake));
+  }
```

Comment:
I know using instant command already takes care of this but I would still prefer `onTrue` over `whileTrue` for this.